src/Tribe/Tickets_Handler.php

```diff
@@ -271,6 +271,121 @@ public function update_start_date( $post_id, $post, $update ) {
 		return true;
 	}
 
+	/**
+	 * The simplest way to grab all the relationships from Any Ticket related objects
+	 *
+	 * On RSVPs Attendees and Orders are the same Post
+	 *
+	 * @param  int|WP_Post  $object  Which object you are trying to figure out
+	 * @return object
+	 *         {
+	 *             'provider' => (mixed|null)
+	 *             'event' => (int|null)
+	 *             'product' => (int|null)
+	 *             'order' => (int|string|null)
+	 *             'order_item' => (int|null)
+	 *         }
+	 */
+	public function get_object_connections( $object ) {
+		// If you add any new Items here, update the Docblock
+		$connections = (object) array(
```

Comment:
Nitpicking here but since in this line you are essentially initializing an empty connections template why not refactor that in a `public` method? E.g. `$connections = $this->get_connections_template();`.
That method would then be available to any client code that needs/wants to know how the empty template looks like.

Reply:
Fair, will do that.